src/privateStore.js

```diff
@@ -43,9 +43,12 @@ class PrivateStore {
   async set (key, value) {
     if (!this.db) throw new Error('_sync must be called before interacting with the store')
 
-    if (value != null) {
-      value = this._encryptEntry(value)
+    if (value == null) {
+      // Do nothing for null value
+      // To remove a value, call remove
+      return false
```

Comment:
Do we really need to check this? I'd just remove it for simplicity. Not really any need to stop people from setting stuff to null.

Reply:
Let me can leave the behavior the way it was regarding setting `null`.